laravel/app/helpers.php

```diff
@@ -2,13 +2,24 @@
 
 namespace App;
 
-class Helpers {
-  public static function displayName($user, $alt=null) {
-    return (!is_null($user) &&
-            !is_null($user->data) &&
+class Helpers
+{
+    /**
+     * display name Helper
+     * Return a burner if the given user has one or return an empty string.
+     * If $alt is false and no burner exists, return the user name of the user
+     * instead.
+     * If $alt is anything else and no burner exists, return the supplied $alt.
+     * @param  User   $user   the user (non-null)
+     * @param  string $alt    the alternative replacement for the burner name
+     * @return string         the appropriate display name
+     */
+    public static function displayName($user, $alt='')
+    {
+        return (!is_null($user->data) &&
```

Comment:
This nested ternary multi-line return statement is a bit difficult to understand at first glance. I'd recommend splitting this up into a couple if statements instead. It'll be a bit longer but much easier to read

Reply:
Very true, I'll be sure to make a more concerted reability pass in future writes.